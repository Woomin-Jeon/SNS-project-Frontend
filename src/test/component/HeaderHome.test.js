import React from 'react';
import {shallow} from "enzyme";
import {BrowserRouter} from "react-router-dom";
import HomeHeader from "../../components/headers/HeaderHome";

describe('HomeHeader', () => {
  let loginState;
  let setLoginState;
  let currentUserState;
  let setCurrentUserState;

  it('renders well', () => {
    const components = shallow(
      <BrowserRouter>
        <HomeHeader
          loginState={loginState}
          setLoginState={setLoginState}
          currentUserState={currentUserState}
          setCurrentUserState={setCurrentUserState}
        />
      </BrowserRouter>
    )

    expect(components).toMatchSnapshot();
  })
})
