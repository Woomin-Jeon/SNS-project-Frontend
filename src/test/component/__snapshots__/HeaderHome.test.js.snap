// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomeHeader renders well 1`] = `ShallowWrapper {}`;
